### PR TITLE
Moved react_module

### PR DESCRIPTION
**Pull request by micah-bryant**
I followed the same type of format used for het_module. Just was wondering about the treatment of dLi because in the het_module for Julia it was put into an if statement which i decided to do the same. Also I noticed that you didn't include the l2_norm function in the het_module function. Is the l2_norm function something you would still like to be moved over and included in het and react module or is it something that was specific to C++?

### CODE REVIEW

**Comment by aarmey** [comment]
Correct for both—dLi is for tracking the endosomal ligand, so the if statement lets the same code be used for surface reactions. l2_norm was for a unit test in the C code that I don't think we need to include. Will review now.

File: src/reactCode.jl [hunk 1]
```diff
@@ -93,6 +93,31 @@ function het_module(Rone, Rtwo, dRone, dRtwo, hetR, hetDim, dhetDim, tr, Gas, dL
...
+                       r.binding[3]*R[2]*Gas - r.binding[4]*R[4],
+                       r.binding[1]*R[3]*Gas - r.binding[2]*R[4],
+                       tr.xFwd*R[1]*R[2] - r.xRev[1]*R[5],
+                       tr.xFwd*R[1]*R[3] - r.xRev[2]*R[5],
+                       tr.xFwd*R[1]*R[4] - r.xRev[3]*R[6],
+                       tr.xFwd*R[2]*R[2] - r.xRev[4]*R[6],
+                       tr.xFwd*R[3]*R[3] - r.xRev[5]*R[6],
+                       r.xFwd6*Gas*R[5] - r.xRev[6]*R[6]])
+    
+    dR[1] += - dRr[7] - dRr[6] - dRr[5] - dRr[1] - dRr[2]; # AXL
```
Comment (aarmey) [inline src/reactCode.jl]: Semicolons aren't necessary in Julia.

Comment (micah-bryant) [reply src/reactCode.jl]: I deleted the semicolons, but is it ok to merge with these failed checks or are these the result of some functions not being transferred over yet?

Comment (aarmey) [reply src/reactCode.jl]: Oh, it's just because the coverage went down. I'll override and merge.

**Pull request merged**